### PR TITLE
Readme updates and Dockerfile updates

### PR DESCRIPTION
**Pull request by grantat**
Updates to Readme and Dockerfile. More suggestions welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ The backlinks calculation is costly to your computers, so it is recommended to t
 $ ./main.py -l search {URI-R} -e cdGetBacklinks
 ```
 
-## How to add your module:
+## How to add your module
 
-Name your module main script as cdGet\<Module name\>.py
+Name your module main script as cdGet\<Module name\>.py. If you module relies on extra code in a different folder it is alright to bring this directory into the modules directory, the carbon tool will ignore subfolders while loading. If the extra code is not in a subfolder, after copying it to the ./modules folder, add the file names into the config file, under 'SystemUtility' field.
```
Comment (ibnesayeed) [inline README.md]: Correct some typos and improve highlights here:

* module main => module's main
* cdGet\<Module name\>.py => `cdGet<ModuleName>.py`
* you module => your module
* carbon tool => CarbonDate library
* while loading => while loading modules
* ./modules folder => `./modules` folder
* 'SystemUtility' => `SystemUtility`

**Review by ibnesayeed — CHANGES_REQUESTED** [review]
Please correct some verbiage as indicated in the inline comment.

**Comment by grantat** [comment]
Changes applied

File: README.md [hunk 2]
```diff
@@ -55,9 +55,9 @@ The backlinks calculation is costly to your computers, so it is recommended to t
 $ ./main.py -l search {URI-R} -e cdGetBacklinks
 ```
 
-## How to add your module:
+## How to add your module
 
-Name your module main script as cdGet\<Module name\>.py
+Name your module's main script as `cdGet\<Module name\>.py`. If your module relies on extra code in a different folder you may bring this directory into the modules directory, the CarbonDate library will ignore subfolders while loading modules. If the extra code is not in a subfolder, after copying it to the `./modules` folder, add the file names into the config file, under `SystemUtility` field.
```
Comment (ibnesayeed) [inline README.md]: `cdGet\<Module name\>.py` => `cdGet<ModuleName>.py` (note the camelCase module name without a space, also escaping is unnecessary.)

Should "under `SystemUtility` field" be "under the `SystemUtility` field"?

**Review by ibnesayeed — CHANGES_REQUESTED** [review]
Some minor wording changes.

**Comment by ibnesayeed** [comment]
LGTM

**Pull request merged**